README.md

```diff
@@ -18,6 +18,37 @@ Now start ZeroTier:
 
 ## Compiling from Sources
 
+### Feed
+A working OpenWRT/LEDE build enviroment is expected (else see below).
+
+Put this line in your feed definition (eg. feeds.conf.default)
+```
+src-git zerotier https://github.com/muebau/zerotier-openwrt.git
```

Comment:
Hi,
yes I was not sure I you would like the code, so I made it fit for my repository. I love to see you used my code and changed it to fit for the public (your repository).
BTW: thank you for your work to provide an OpenWRT package.